scipy/stats/_continuous_distns.py

```diff
@@ -8501,7 +8504,18 @@ def _updated_ctor_param(self):
         dct = super(rv_histogram, self)._updated_ctor_param()
         dct['histogram'] = self._histogram
         return dct
-
+    
+    def fit( data , **kwds ):
```

Comment:
looks like a few other `fit()` methods implemented for subclasses of `rv_continuous` have a call signature like this:

`fit(self, data, *args, **kwds):`

It looks like the unit test machinery that iterates over distributions for testing is expecting a signature like that and causing a failure here.